generate_commitment/generate_submarine_commit.py

```diff
@@ -49,19 +45,22 @@ def generateRS(addressA , addressC , sendAmount , dappData, gasPrice, gasLimit):
 
     '''
     #TODO: validate AddressA and AddressC
-    commit, randw = generateCommit(addressA, addressC, sendAmount, dappData, gasPrice, gasLimit)
+    commit, randw = generateCommit(addressA, addressC, sendAmount, dappData,
+                                   gasPrice, gasLimit)
 
     R = bytearray_to_int(sha3_256(commit + bytes(1)))
     S = bytearray_to_int(sha3_256(commit + bytes(0)))
 
-    if (0 < R < secp256k1n) & (0 < S < (secp256k1n/2)):
+    if (0 < R < secp256k1n) & (0 < S < (secp256k1n / 2)):
         return commit, randw, R, S
     else:
         log.info("Invalid R,S. Regenerating the hashes...")
-        return generateRS(addressA, addressC, sendAmount, dappData, gasPrice, gasLimit)
+        return generateRS(addressA, addressC, sendAmount, dappData, gasPrice,
+                          gasLimit)
 
 
-def generateCommit(addressA , addressC , sendAmount , dappData, gasPrice, gasLimit):
+def generateCommit(addressA, addressC, sendAmount, dappData, gasPrice,
```

Comment:
Do we use `_` in front of internal functions?

Reply:
that, according to my understanding, is the pythonic way of notating internal functions